tutorials/introduction-to-ripqp/index.jmd

```diff
@@ -95,7 +95,7 @@ At every iteration, RipQP solves two linear systems with the default Predictor-C
 
 To save these systems, you can use:
 
-```julia
+```
```

Comment:
What is the motivation for this? It feels weird to have block of code that are not executed.